Libraries/src/Amazon.Lambda.AspNetCoreServer/Internal/Utilities.cs

```diff
@@ -18,6 +20,45 @@ namespace Amazon.Lambda.AspNetCoreServer.Internal
     /// </summary>
     public static class Utilities
     {
+
+        internal static void EnsureLambdaServerRegistered(IServiceCollection services)
+        {
+            IList<ServiceDescriptor> toRemove = new List<ServiceDescriptor>();
+            var serviceDescriptions = services.Where(x => x.ServiceType == typeof(IServer));
+            int lambdaServiceCount = 0;
+
+            // There can be more then one IServer implementation registered if the consumer called ConfigureWebHostDefaults in the Init override for the IHostBuilder.
+            // This makes sure there is only one registered IServer using LambdaServer and removes any other registrations.
+            foreach (var serviceDescription in serviceDescriptions)
+            {
+                // If Lambda server has already been added the skip out.
```

Comment:
the skip out => then skip out

Reply:
Actually the comment was out of date from when this code assumed there would only be one registration of an IServer. I removed the comment and added an additional comment below when removing a registration.